src/pages/lessons/[slug].tsx

```diff
@@ -236,9 +264,16 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
   }
 
   const completeVideo = (lessonView: any) => {
+    console.debug('completed video', {lessonView, video: lesson})
     if (lessonView) {
       const hasNextLesson = nextLesson
       const progress = getProgress(lessonView)
+
+      if (!hasNextLesson && isFullscreen) {
```

Comment:
if we are fullscreenin and there's no more videos to watch, jump out of fullscreen